src/ImagePotentialBond.h

```diff
@@ -0,0 +1,249 @@
+                }
+
+            // add the force to the particles (only for non-ghost particles)
+            if (idx_b < this->m_pdata->getN())
+                {
+                h_force.data[idx_b].x += force_divr * dx.x;
+                h_force.data[idx_b].y += force_divr * dx.y;
+                h_force.data[idx_b].z += force_divr * dx.z;
+                h_force.data[idx_b].w += bond_eng;
+                if (compute_virial)
+                    for (unsigned int i = 0; i < 6; i++)
+                        h_virial.data[i * this->m_virial_pitch + idx_b] += bond_virial[i];
+                }
+
+            if (idx_a < this->m_pdata->getN())
+                {
+                h_force.data[idx_a].x -= force_divr * dx.x;
+                h_force.data[idx_a].y -= force_divr * dx.y;
+                h_force.data[idx_a].z -= force_divr * dx.z;
+                h_force.data[idx_a].w += bond_eng;
+                if (compute_virial)
+                    for (unsigned int i = 0; i < 6; i++)
+                        h_virial.data[i * this->m_virial_pitch + idx_a] += bond_virial[i];
+                }
+            }
+        else
+            {
+            this->m_exec_conf->msg->error()
+                << "bond." << evaluator::getName() << ": bond out of bounds" << std::endl
+                << std::endl;
+            throw std::runtime_error("Error in bond calculation");
+            }
+        }
+    }
+
+#ifdef ENABLE_MPI
+/*! \param timestep Current time step
+ */
+template<class evaluator, class Bonds>
+CommFlags ImagePotentialBond<evaluator, Bonds>::getRequestedCommFlags(uint64_t timestep)
+    {
+    CommFlags flags = md::PotentialBond<evaluator, Bonds>::getRequestedCommFlags(timestep);
+    flags[comm_flag::image] = 1;
+
+    return flags;
+    }
+#endif
+
+namespace detail
+    {
+//! Exports the ImagePotentialBond class to python
+/*! \param name Name of the class in the exported python module
+    \tparam T Evaluator type to export.
+*/
+template<class T> void export_ImagePotentialBond(pybind11::module& m, const std::string& name)
+    {
+    pybind11::class_<ImagePotentialBond<T, BondData>,
+                     md::PotentialBond<T, BondData>,
+                     std::shared_ptr<ImagePotentialBond<T, BondData>>>(m, name.c_str())
+        .def(pybind11::init<std::shared_ptr<SystemDefinition>>());
+    }
+
+//! Exports the PotentialMeshBond class to python
+/*! \param name Name of the class in the exported python module
+    \tparam T Evaluator type to export.
+*/
+template<class T> void export_PotentialMeshBond(pybind11::module& m, const std::string& name)
+    {
+    pybind11::class_<ImagePotentialBond<T, MeshBondData>,
+                     md::PotentialBond<T, MeshBondData>,
+                     std::shared_ptr<ImagePotentialBond<T, MeshBondData>>>(m, name.c_str())
+        .def(pybind11::init<std::shared_ptr<SystemDefinition>, std::shared_ptr<MeshDefinition>>());
+    }
```

Comment:
I don't think we have any support for meshes in azplugins, so we can just remove this export method (which is unused).
```suggestion
```